state_transfer/endpoint/route/route.go

```diff
@@ -76,80 +80,103 @@ func (r *RouteEndpoint) Labels() map[string]string {
 	return r.labels
 }
 
-func (r *RouteEndpoint) Type() endpoint.EndpointType {
+func (r *RouteEndpoint) Type() RouteEndpointType {
```

Comment:
Why can't they use reflection to get this value? wonder if we can remove this

Reply:
Programmer error, this is supposed to be consumed internally, not supposed to be exported.

Reply:
removed